templates/zammad/0/rancher-compose.yml

```diff
@@ -0,0 +1,29 @@
+.catalog:
+  name: "Zammad"
+  version: "1.2"
+  description: "Zammad is a web based open source helpdesk/customer support system"
+  uuid: zammad-1
+  minimum_rancher_version: v1.2.0
+  questions:
```

Comment:
# Add service combo question to select elasticsearch service. You need to deploy it before.
    - variable: "zammad_elasticsearch_service"
      description: |
        Choose the elasticsearch service to use. 
      label: "Elasticsearch stack/service"
      default: "elasticsearch-2/kopf"
      required: true
      type: "service"